distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerProxy.java

```diff
@@ -38,8 +42,15 @@ public ServiceManagerProxy(ServiceManager serviceManager) {
 
     @Override
     public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
-
-        Subject subject = SECURITY.runAsAdmin(SECURITY::getSystemSubject);
-        return subject.execute(() -> method.invoke(serviceManager, args));
+        final RetryPolicy retryPolicy = new RetryPolicy().retryOn(NullPointerException.class)
```

Comment:
@clockard added this retry in a separate PR now that this error is occurring in itests even without the pax exam changes. I have rebased this PR from https://github.com/codice/ddf/pull/2015.